great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -156,6 +155,17 @@ def run(
         expectation_suite_name: Optional[str] = None,
         include_citation: bool = True,
     ) -> DataAssistantResult:
+        """
+        Run the DataAssistant as it is currently configured.
+
+        Args:
+            expectation_suite: An existing "ExpectationSuite" to update.
+            expectation_suite_name: A name for returned "ExpectationSuite".
```

Comment:
@NathanFarmer periods to be removed for consistency?

Reply:
✅